lib/doorkeeper/custom_token_response.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Doorkeeper
+  module CustomTokenResponse
+    def body
+      {
+        data: {
+          id: @token.id,
+          type: :token,
```

Comment:
Why is the `type` a symbol `:token` and not a string "token"?

Reply:
I just prefer using symbol than string in this case, not sure there are any problems here.
I'll change back

Reply:
Not sure if there is a problem either, just wondering why 😆